app/javascript/dashboard/helper/directives/resize.js

```diff
@@ -0,0 +1,41 @@
+import { debounce } from '@chatwoot/utils';
```

Comment:
I have taken resize directives from this [commit](https://github.com/chatwoot/chatwoot/pull/9109/commits/af90e3a56cff138db408a698d1a5a21b69072263)